internal/k8shandler/collection.go

```diff
@@ -93,6 +95,18 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 		return err
 	}
 
+	var httpInputs []string
```

Comment:
I would move this newly added code into ReconcileInputServices(), since what it is doing is reconciling input services and as it is, it breaks the pattern in this function of a sequence of Reconcile<T>.

Reply:
This wasn't new code, I just moved it around as it was later in this function. However, I have now moved this to where I think the appropriate location should be.